test-lib/app/com/commercetools/sunrise/it/WithSphereClient.java

```diff
@@ -54,4 +59,10 @@ protected static SphereClientConfig sphereClientConfig() {
         final String clientSecret = configuration.getString("ctp.it.clientSecret");
         return SphereClientConfig.of(projectKey, clientId, clientSecret);
     }
+
+    protected static HttpClient newHttpClient() {
```

Comment:
Why do we need this method here? If we need it for a particular test, I would leave it on that place. I wouldn't polute this class which is meant as a helper class for developers.